lib/login/login_page.dart

```diff
@@ -8,10 +9,106 @@ class LoginPage extends StatelessWidget {
+            children: [
+              Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 40),
+                child: TextField(
+                  obscureText: true,
+                  decoration: InputDecoration(
+                    icon: Icon(
+                      Icons.person,
+                      color: MyPlantColors.lightGrey,
+                    ),
+                    hintText: 'Username',
+                    hintStyle: TextStyle(
+                      color: MyPlantColors.lightGrey,
+                    ),
+                    enabledBorder: myUnderlineBorder(),
+                  ),
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(
+                  bottom: 40,
+                ),
+                child: TextField(
+                  obscureText: true,
+                  decoration: InputDecoration(
+                    icon: Icon(
+                      Icons.lock,
+                      color: MyPlantColors.lightGrey,
+                    ),
+                    hintText: 'Password',
+                    hintStyle: TextStyle(
+                      color: MyPlantColors.lightGrey,
+                    ),
+                    enabledBorder: myUnderlineBorder(),
+                  ),
+                ),
+              ),
+              LoginButton(
+                onLoginClicked: () {},
+              ),
+              Padding(
+                padding: const EdgeInsets.only(
+                  top: 40,
+                ),
+                child: Row(
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.only(
+                        right: 60,
+                      ),
+                      child: TextButton(
+                        onPressed: () {},
+                        child: Text(
+                          'Forgot Password ?',
+                          style: TextStyle(
+                            fontSize: 12,
+                            fontWeight: FontWeight.normal,
+                            color: MyPlantColors.greenSmoke.withOpacity(.7),
+                          ),
+                        ),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.only(
+                        left: 60,
+                      ),
+                      child: TextButton(
+                        onPressed: () {},
+                        child: Text(
+                          'Sign Up',
+                          style: TextStyle(
+                            fontSize: 12,
+                            fontWeight: FontWeight.normal,
+                            color: MyPlantColors.greenSmoke.withOpacity(.7),
+                          ),
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            ],
+          ),
```

Comment:
You are using Padding-Widget to set space between the widgets in the column. Maybe, it would be better, if you use the SizedBox widget with a specific height oder width. And it would be make sense to outsource this SizedBox in case of multiple occurrence.